### ISSUE TITLE
Notes/comment on a list item

### ISSUE THREAD

**Issue by skillix** [Apollon77/alexa-remote#240]
Hello, I noticed that in shopping lists, todo lists, or simple lists, it is possible to add a note to an item on the list.

![IMG_20240411_151746](https://github.com/Apollon77/alexa-remote/assets/68369861/2c9f0ebc-8cee-42be-9643-c05c4b853ba3)
![IMG_20240411_151814](https://github.com/Apollon77/alexa-remote/assets/68369861/90c10778-c86a-4949-8129-14138300c9e9)

Via your API I cannot find these notes.

Logically as it is possible to do it via the application, it must be technically possible.

Did I miss an option, if you have an idea to get them and write them down that would be great.

I also noticed that alongside the lists there are simple notes, and I don't seem to be able to retrieve them via your API either, is this possible?

![IMG_20240411_151713](https://github.com/Apollon77/alexa-remote/assets/68369861/3e557006-93e3-4eaf-aa09-7ea00889b32b)

Ps: I can't reduce the size of the images

**Comment by Apollon77** [comment]
Seems they added that as new feature in the app, so yes technically possiblebut requires investigation and new API reverse engineering. You can try yourself and provide a PR, else when I find time

**Comment by skillix** [comment]
my sniffing software no longer works with the new android versions, do you have any advice?

**Comment by Apollon77** [comment]
Ok seems there is a new lists v2 API ....

**Comment by padrino-x** [comment]
@skillix 
Are you sniffing the app?
Is there an available Software to use?
Personally I use an extremely old version of the app, ie. 2.2.466191.0.
Maybe that could be used for sniffing?
Yes, all those changes for notes work with that old version, too.

**Comment by Apollon77** [comment]
I use Charles Proxy. Then you import the SSL cert (https://www.charlesproxy.com/documentation/using-charles/ssl-certificates/)  and then you setup the proxy. That is all much easier on ios btw ... android with the certs is a bit difficult

**Comment by Apollon77** [comment]
So basically "notes" are already possible. The further "comments" not yet

**Comment by Apollon77** [comment]
No, fact fornow :-) But I know how the "memories" work ... "just" need to add them
